.github/workflows/build.yml

```diff
@@ -0,0 +1,22 @@
+name: Run ESLint
```

Comment:
I see that the name is RunEslint, but it looks like this workflow does more than that the name could be changed here.

 This install deps and runs builds the app on every push and on every branch, is that the intention or do you only want this action to happen on the mainline branch? that can be configured here.